src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -71,7 +79,9 @@ export function Breadcrumbs({ list, Component }: BreadcrumbsListProps) {
             {Component ? (
               <Component to={br.uri || ""}>{br.label}</Component>
             ) : (
-              <a href={br.uri || ""}>{br.label}</a>
+              <StyledLink isDisabled={br.isDisabled} href={br.uri || ""}>
```

Comment:
`br.uri || ""` is redundant (looking at the type. `Uri` can be only of `string` value.